src/clj_github/repository.clj

```diff
@@ -5,7 +5,8 @@
             [clojure.java.io :as io]
             [clojure.string :as string]
             [me.raynes.fs :as fs]
-            [me.raynes.fs.compression :as fs-compression])
+            [me.raynes.fs.compression :as fs-compression]
+            [ring.util.codec :as encode])
```

Comment:
As described in the [clojure-style-guide](https://github.com/nubank/clojure-style-guide#namespaces), we should use the alias the same as the namespace name:

```suggestion
            [ring.util.codec :as codec])
```